client-js/app/index.html

```diff
@@ -43,9 +43,11 @@
        websocket api connections should be made. Comments are stripped
        when we built the dist version, so this has no effect in prod. -->
   <!-- DEV_MODE_CONFIG -->
+
+  <link rel="import" href="/bundle.html">
 </head>
 
 <body unresolved class="fullbleed layout vertical">
-  <script src="scripts/bundle.js"></script>
+  <span id="browser-sync-binding"></span>
```

Comment:
Can this be a comment instead of an actual html element? We only need browser sync in dev mode, and comments get stripped out when we build for production mode.